### DISCUSSION TITLE
How do I add a environment variable to run in the sandbox?

### DISCUSSION THREAD

**Discussion by dhidary** [saleweaver/python-amazon-sp-api#196] [Q&A] [answered]
I'm not sure what to do with "export AWS_ENV="SANDBOX" "
Thanks

**Comment by Zakir-Ayub** [comment]
Facing same problem, No idea what to do with this, some clarifications needed.

**Comment by saleweaver** [comment, marked as answer]
If you want to run a request against the api's sandbox, you can use the environment variable. For example, to run all tests, use

AWS_ENV=SANDBOX pytest

as command.

**Reply by michalaleksandrowicz** [reply]
hey, I still struggle on how to run the sandbox, where do I put the "AWS_ENV=SANDBOX pytest" command? in the script?
